### ISSUE TITLE
pyLCIO issue with ROOT 6.20

### ISSUE THREAD

**Issue by petricm** [iLCSoft/LCIO#76]
I just updated the CI to LCG_97 and the pyLCIO tests fails with

```
Loading LCIO ROOT dictionaries ...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Package/src/python/pylcio.py", line 3, in <module>
    from pyLCIO.io.EventLoop import EventLoop
  File "/Package/src/python/pyLCIO/__init__.py", line 16, in <module>
    from .base.HandleExceptions import setupExceptionHandling
  File "/cvmfs/sft.cern.ch/lcg/views/LCG_97/x86_64-centos7-gcc9-opt/lib/ROOT.py", line 522, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/Package/src/python/pyLCIO/base/HandleExceptions.py", line 17, in <module>
    from libcppyy import CPPOverload
  File "/cvmfs/sft.cern.ch/lcg/views/LCG_97/x86_64-centos7-gcc9-opt/lib/ROOT.py", line 522, in _importhook
    return _orig_ihook( name, *args, **kwds )
ImportError: No module named libcppyy
```

**Comment by petricm** [comment]
Thisi is just in LCG_97(ROOT 6.20) and not LCG_96b (ROOT 6.18.04)

**Comment by andresailer** [comment]
Should be fixed by #75
